src/luarocks/util.lua

```diff
@@ -454,7 +454,7 @@ do
 
       local ljv
       if cfg.lua_version == "5.1" then
-         ljv = util.popen_read(Q(cfg.variables["LUA_BINDIR"] .. "/" .. cfg.lua_interpreter) .. ' -e "io.write(tostring(jit and jit.version:gsub([[^%w-JIT ]], [[]])))"')
+         ljv = util.popen_read(Q(cfg.variables["LUA_BINDIR"] .. "/" .. cfg.lua_interpreter) .. ' -e "io.write(tostring(jit and jit.version:gsub([[^%w+ ]], [[]])))"')
```

Comment:
Does this work? `%w` is alphanumeric and won't include `-`, `_` and any other punctuation we may expect here

Reply:
It will for now, while we don't have another LuaJIT fork with dashes :) But `%S` should work too.

(I clicked "resolved" by accident from the phone and apparently Github's mobile site doesn't include "unresolve", sorry about that)